src/renderers/dom/client/utils/setInnerHTML.js

```diff
@@ -28,7 +31,20 @@ var createMicrosoftUnsafeLocalFunction = require('createMicrosoftUnsafeLocalFunc
  */
 var setInnerHTML = createMicrosoftUnsafeLocalFunction(
   function(node, html) {
-    node.innerHTML = html;
+    if ('innerHTML' in node) {
```

Comment:
Yeah let's do…

```
if (node.namespaceURI === DOMNamespaces.svg && !('innerHTML' in node)) {
  // workaround
} else {
  node.innerHTML = html;
}
```

Then I think this looks good if you have done manual testing that reproduces the original issue and confirms this fixes it.